components/nif/nifkey.hpp

```diff
@@ -50,6 +50,12 @@ struct KeyMapT {
 
     KeyMapT() : mInterpolationType(sLinearInterpolation) {}
 
+    // For serializer.
```

Comment:
The built in serializer sometimes _requires_ getX and setX methods to exist with very specific declarations.  To violate this convention requires hand-coding a lot of extra stuff. :-(  For this version, the primary goal was just to get it working.   Is this a problem?